src/codemagic/firebase/resources/release.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from dataclasses import field
+from datetime import datetime
+from datetime import timezone
+
+from .abstract_resource import AbstractResource
+
+
+@dataclass
+class ReleaseResource(AbstractResource):
+    """
+    https://firebase.google.com/docs/reference/app-distribution/rest/v1/projects.apps.releases#ReleaseNotes
+    """
+    @dataclass
+    class ReleaseNotes:
+        text: str
```

Comment:
Why not declare it at top level and have a docstring for the class that points to Firebase docs?